SPECS-EXTENDED/liblouis/liblouis.spec

```
@@ -0,0 +1,407 @@
+# Turn off the brp-python-bytecompile script
+%global __os_install_post %(echo '%{__os_install_post}' | sed -e 's!/usr/lib[^[:space:]]*/brp-python-bytecompile[[:space:]].*$!!g')
+Summary:        Braille translation and back-translation library
+Name:           liblouis
+Version:        3.22.0
+Release:        2%{?dist}
+License:        LGPLv3+
```

Small nit: We're trying to slowly change these license tags to use [SPDX expressions](https://spdx.org/licenses/). Mind using the SPDX name for this and other licenses?

```suggestion
License:        LGPL-3.0-or-later
```

---

Thanks for suggesting this. I have modified the license tags as per SPDX expressions